auth/client-provider/src/test/resources/org/wildfly/security/auth/client/provider/test-wildfly-config-client-default-sslcontext.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+    <authentication-client xmlns="urn:elytron:client:1.5">
+        <key-stores>
+            <key-store name="keystore1" type="JKS">
+                <file name="src/test/resources/client.keystore"/>
```

Comment:
Let me know if I should generate this keystore instead. It is not actually used to connect in the test I just needed some path here. The client.keystore is copied from [test folder of auth/client.](https://github.com/wildfly-security/wildfly-elytron/blob/1.x/auth/client/src/test/resources/client.keystore)

Reply:
Just to keep things simpler, if it is possible to generate it instead that would be a cleaner approach.